Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM cosmwasm/wabt:1.0.25 as wabt
 
-FROM cosmwasm/tinygo:0.19.3 as tinygo
+# Using the newest CosmWasm TinyGo v0.23.0 beta (stable version: cosmwasm/tinygo:0.19.3)
+FROM cosmwasm/tinygo:cw-0.23.0 as tinygo
```

Comment:
I would just call this `cosmwasm/tinygo:0.23.0`

All docker tags on `cosmwasm/tinygo` are cw-specific

Reply:
I was following [this](https://github.com/CosmWasm/tinygo/tags). Should I re-release the TinyGo with `0.23.0` instead of `cw-0.23.0` or leave it as it is for now?

Reply:
And one more thing. You can use CW TinyGo repo to build a non-CW app, but the `cw-...` can ONLY be used for CosmWasm (other builds would fail). That was an other idea behind the naming.

Reply:
The `cw-...` Docker image simply doesn't have all the dependencies for other platforms to make the build faster (though it takes 1.5 with a default GH actions node 🤷‍♂️).

Reply:
1.5 hours builds?? And I thought those 10-15 minutes on my laptop were bad.

Reply:
> Should I re-release the TinyGo with 0.23.0 instead of cw-0.23.0 or leave it as it is for now?

Yeah, I wouldn't match the tags. I would use the official `tinygo/tinygo` image for the defaults, and only publish the cosmwasm feature flag.

No problem with the cw-0.23.0 tag. I would just add the same as 0.23.0 as well, and do future builds as 0.23.1, etc